modelangelo/__init__.py

```diff
@@ -67,10 +57,13 @@ def getModelAngeloCmd(cls):
 
     @classmethod
     def getEnviron(cls):
-        environ = pyworkflow.utils.Environ(os.environ)
+        environ = pwutils.Environ(os.environ)
         torch_home = cls.getVar(TORCH_HOME_VAR)
-        # For GPU, we need to add to LD_LIBRARY_PATH the path to Cuda/lib
         environ.set(TORCH_HOME_VAR, torch_home)
+
+        cudaLib = cls.getVar(MODEL_ANGELO_CUDA_LIB, pwem.Config.CUDA_LIB)
```

Comment:
Default value here, although is does not harm is not necessary. Don't need to change it.

Reply:
not sure what you mean... you want to remove this line?

Reply:
Ah!! of course!